src/components/FileRepoSidebar.js

```diff
@@ -30,6 +37,17 @@ const styles = {
   `,
 };
 
+const getGen3UUIDs = async arrangerIds => {
+  const fileData = await getFilesById({ ids: arrangerIds, fields: ['uuid'] });
+  return fileData.map(file => file.node.uuid);
+};
+
+const downloadFile = async (arrangerIds, gen3Key) => {
+  let files = await getGen3UUIDs(arrangerIds);
+  console.log(files);
+  let fileUUID = files && files.length > 0 ? files[0] : '';
+  return downloadFileFromGen3(gen3Key, fileUUID);
```

Comment:
does it make sense to still call this function with an empty string for `fileUUID`?